src/status_im/ui/components/list_selection.cljs

```diff
@@ -1,62 +1,50 @@
 (ns status-im.ui.components.list-selection
   (:require [re-frame.core :refer [dispatch]]
-            [status-im.ui.components.react :refer [copy-to-clipboard
-                                                   sharing
-                                                   linking]]
+            [status-im.ui.components.react :as react]
             [status-im.utils.platform :refer [platform-specific ios?]]
             [status-im.i18n :refer [label]]))
 
-(defn open-share [content]
+(defn- open-share [content]
   (when (or (:message content)
             (:url content))
-    (.share sharing (clj->js content))))
+    (.share react/sharing (clj->js content))))
 
 (defn share-options [text]
-  [{:text  (label :t/sharing-copy-to-clipboard)
-    :value #(copy-to-clipboard text)}
-   {:text  (label :t/sharing-share)
-    :value #(open-share {:message text})}])
+  [{:label  (label :t/sharing-copy-to-clipboard)
+    :action #(react/copy-to-clipboard text)}
+   {:label  (label :t/sharing-share)
+    :action #(open-share {:message text})}])
+
+(defn show [options]
+  ((:list-selection-fn platform-specific) options))
```

Comment:
maybe better to use dialog component? `:list-selection-fn` looks weird

Reply:
Sure! I didn't fully finished the refactoring for some reason .. ;)